NHLBI_BioData_Catalyst/Rstudio/1_PICSURE_API_101.Rmd

```diff
@@ -186,10 +179,10 @@ if(nrow(my_search_df) == 0){
 ## Using PIC-SURE to build a query and retrieve data
 You can also use the PIC-SURE API to build a query and retrieve data. With this functionality, you can filter based on specific variables, add others, and export the data as a dataframe into this notebook. 
 
-The first step to this is setting up the `authQuery`.
+The first step is to set up the query object.
 
 ```{r}
-authQuery_categorical_example <- bdc::new.query(authPicSure)
+query_categorical_example <- picsure::bdc.newQuery(session)
 ```
 
 There are several methods that can be used to build a query, which are listed below.
```

Comment:
The following in-line documentation needs to be updated for the new functions

| Method | Arguments / Input | Output|
|--------|-------------------|-------|
| query.select.add() | variable names (string) or list of strings | all variables included in the list (no record subsetting)|
| query.require.add() | variable names (string) or list of strings | all variables; only records that do not contain null values for input variables |
| query.anyof.add() | variable names (string) or list of strings | all variables; only records that contain at least one non-null value for input variables |
| query.filter.add() | variable name and additional filtering values | input variable; only records that match filter criteria |